Reply/app/src/main/res/layout/search_suggestion_item.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright (c) 2020 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+  in compliance with the License. You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software distributed under the License
+  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+  or implied. See the License for the specific language governing permissions and limitations under
+  the License.
+  -->
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+        <variable
+            name="suggestion"
+            type="com.materialstudies.reply.data.SearchSuggestion" />
+    </data>
+
+    <LinearLayout
```

Comment:
Any thoughts on replacing this with a constraint layout? Would it be cleaner/less nested?

Reply:
fixed! definitely cleaner (also copied layout from catalog where we can't really use constraintlayout as easily 😊)